AletheiaFact/aletheia
# 1505 - Migrate Button component from Ant Design to MUI

# Description
*Button.tsx: Migrate Button component from Ant Design to Material-UI
- Replaced Ant Design's Button with Material-UI's Button in the project.
- Updated all references to AntD's ButtonProps and related APIs.
- Introduced `buttonType` to map custom button styles previously managed with AntD's `type`.
- Replaced `htmlType` with `type` for handling button functionality (`submit`, `reset`, `button`).
- Updated usage of `loading`, `icon`, and other AntD-specific props to Material-UI equivalents.
- Ensured compatibility with href, target, and style props in the new implementation.*

Fixes #1505 

# Type of change
*Please delete options that are not relevant.*


# Testing
*1. Be logged into the platform;
2. Click on any button within the platform; and
3. See if you are redirected to the correct page that the button corresponds to.*


# Developer Checklist
## General

## Frontend Changes

### Tests

# Merge Request Review Checklist
